C:\Users\test0\source\repos\mython\mython\Boolean.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Boolean.obj
C:\Users\test0\source\repos\mython\mython\Functions.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Functions.obj
C:\Users\test0\source\repos\mython\mython\Helper.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Helper.obj
C:\Users\test0\source\repos\mython\mython\IndentationException.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\IndentationException.obj
C:\Users\test0\source\repos\mython\mython\Integer.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Integer.obj
C:\Users\test0\source\repos\mython\mython\Interpreter.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Interpreter.obj
C:\Users\test0\source\repos\mython\mython\List.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\List.obj
C:\Users\test0\source\repos\mython\mython\NameErrorException.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\NameErrorException.obj
C:\Users\test0\source\repos\mython\mython\Parser.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Parser.obj
C:\Users\test0\source\repos\mython\mython\Sequence.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Sequence.obj
C:\Users\test0\source\repos\mython\mython\Set.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Set.obj
C:\Users\test0\source\repos\mython\mython\String.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\String.obj
C:\Users\test0\source\repos\mython\mython\SyntaxException.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\SyntaxException.obj
C:\Users\test0\source\repos\mython\mython\Type.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Type.obj
C:\Users\test0\source\repos\mython\mython\Void.cpp;C:\Users\test0\source\repos\mython\mython\x64\Debug\Void.obj
